tradesman/model.py

```diff
@@ -186,6 +187,14 @@ def __initialize_model(self):
             self._project.new(self.__folder)
             add_new_tables(self._project.conn)
 
+    def __country_identifier(self):
+
+        country = self._project.conn.execute(
```

Comment:
If the query is empty, this will fail, since you cannot take the first element of it.

Reply:
I've made some changes in the code to overcome this issue. Now I'm saving some project information using AequilibraE's About module. It probably will be less error-prone.